docs/source/conf.py

```diff
@@ -606,29 +606,22 @@
     # torch.distributed.optim.utils
     "as_functional_optim",
     "register_functional_optim",
-    # torch.distributed.pipeline.sync.checkpoint
```

Comment:
Please delete the corresponding entries! Not just the comment saying which module they come from!

Reply:
hahahaha, sorry my bad, I thought those are real entries, but just commented out :)